index.test.js

```diff
@@ -1,6 +1,6 @@
-const stylelint = require('stylelint');
+import stylelint from 'stylelint';
 
-const config = require('.');
+import config from './index';
```

Comment:
`./index.js`?

https://nodejs.org/api/esm.html#import-specifiers

Reply:
Is this required as we now have an `exports` field in the package.json?

Reply:
> Including the file extension is only necessary for packages without an ["exports"](https://nodejs.org/api/packages.html#exports) field.